src/Currencies/BitcoinCurrencies.php

```diff
@@ -26,6 +27,10 @@ public function contains(Currency $currency)
      */
     public function subunitFor(Currency $currency)
     {
+        if ($currency->getCode() !== self::CODE) {
+            throw new UnknownCurrencyException($currency->getCode().' cannot be found within BitcoinCurrencies');
```

Comment:
I would rather say "XXX is not bitcoin and is not supported by this currency repository"